src/FINDOLOGIC/Export/Helpers/UsergroupAwareSimpleValue.php

```diff
@@ -27,7 +27,7 @@ public function getValues()
 
     public function setValue($value, $usergroup = '')
     {
-        $this->values[$usergroup] = $value;
+        $this->values[$usergroup] = DataHelper::emptyValueCheck($value);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done